src/server/routes/calendar.js

```diff
@@ -5,7 +5,8 @@ const express   = require('express'),
       gcal      = require('google-calendar'),
       jstz      = require('jstz'), // Automatically detect timezone and initialize
       timezone  = jstz.determine();
-
+      Event     = require('../models/event');
```

Comment:
Add 'use strict' (do that) to the top of your file and it will break your code.

Why?

Well because you are not following Flavio's fancy pattern of comma separated declarations like so
```
const num1 = 1,
          num2 = 2,
          string = "We are all declared as const at once because of the commas"

````
Because they are comma separated they belong to the same declaration. Since `timezone` ends in a `;` that declaration of const ends and your variables are actually not explicitly declared so imposing `'use strict'` will break the code.

Please fix and ask Flavio if you don't understand this declaration pattern as he is using it to simply save on writing const over and over. Or feel free to add in new declarations starting with the full const/var declaration again. Just fix.